src/adapters/PeopleJSONAdapter.js

```diff
@@ -0,0 +1,20 @@
+import {Observable} from 'rxjs';
+
+import person from './../data/person';
+import PeopleAdapter from './PeopleAdapter';
+
+export default class PeopleJSONAdapter extends PeopleAdapter {
+  /**
+   * Returns an observable that emits person data
+   *
+   * @param {String} id - ID of person to get
+   * @returns {Observable<PersonObject>}
+   * @memberof PeopleAdapter
+   */
+  getPerson(id) {
+    return new Observable((observer) => {
```

Comment:
I think we should be using the static method [`create`](http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#static-method-create) instead. It will keep it consistent with other types of ways we will be creating observables (e.g. promises).
See https://stackoverflow.com/a/45912937
```suggestion
    return Observable.create((observer) => {
```